project-words.txt

```diff
@@ -699,4 +699,9 @@ redcircle
 bluecircle
 tatarot
 Tatarot
-Scry
\ No newline at end of file
+Scry
```

Comment:
this is causing merge conflicts
add them in alphabetical order at the top to avoid

Reply:
@corbinpage

Reply:
nvm we need these changes in an hour so I just copied them and merged in my own PR

Reply:
Thanks Emily! Apreciate the help!